src/main/java/io/confluent/connect/jdbc/sink/JdbcSinkConfig.java

```diff
@@ -262,6 +267,26 @@ public enum DateTimezone {
       + "the timezone set for db.timzeone configuration (to maintain backward compatibility). It "
       + "is recommended to set this to UTC to avoid conversion for DATE type values.";
 
+  public static final String TIMESTAMP_PRECISION_MODE_CONFIG = "timestamp.precision.mode";
+  public static final String TIMESTAMP_PRECISION_MODE_DEFAULT =
+      TimestampPrecisionMode.MICROSECONDS.toString();
+  private static final String TIMESTAMP_PRECISION_MODE_CONFIG_DISPLAY =
+      "Sink Timestamp Precision mode";
+  private static final String TIMESTAMP_PRECISION_MODE_CONFIG_DOC =
+      "Convert the Timestamp with precision. If set to microseconds, "
+          + "the timestamp will be converted to microsecond precision. If set to "
+          + "nanoseconds the timestamp will be converted to nanoseconds precision.";
+
+  public static final String TIMESTAMP_FIELDS_WHITELIST = "timestamp.conversion.fields";
```

Comment:
nit: Let's also update the variable name.
```suggestion
  public static final String TIMESTAMP_CONVERSION_FIELDS = "timestamp.conversion.fields";
```